### PR TITLE
Loginpage

### PR DESCRIPTION
**Pull request by pranati10**
updated login page
changes made in firebase 
added comments
update readme

### CODE REVIEW

File: src/App.js
Comment (EZRA-DVLPR) [inline src/App.js:1]: Why have the SignIn Component if you're not going to use it?

File: src/firebase.js
Comment (EZRA-DVLPR) [inline src/firebase.js:1]: You still haven't addressed the security concern about these keys being available to the public.

**Comment by EZRA-DVLPR** [comment]
Several Reasons as to why I am not merging this pull request:

1) This is still a different project. It uses an entirely different framework. I already explained this before in the previous pull request rejection. You need to use the base code that is live on main before you add or delete any lines of code. It is nonsensical for me (or anyone) to accept code that is not even on the same project.

2) I didn't mention this in the last PR because I assumed you would resolve it, but there is no need for unnecessary image files like `logo192.png` and `logo512.png`. They aren't in any way associated with the project on main. They aren't harmful to have, but aren't necessary for the website to function. So It'd be best to delete them. It isn't a major concern compared to the other issues regarding your code.

3) Your README still contains information that is indicative of the process of merging. This is noted by the <<<<<<< HEAD line on line 1. This is simply leftover code that needs to be cleaned up if you intend to add anything to the README.

4) You still haven't addressed the security concern about the information being accessed for firebase. This information should not be available to the public. Create a private file and share that through a secure medium after you change these keys. Additionally, these are the same keys and access information indicating you haven't addressed the already existing security issue.

5) If you could explain the purpose of the file `src/reportWebViutals.js` that would be nice. As of right now, I don't understand it's purpose. It doesn't seem to be utilized in any js file. I acknowledge its existence in the package.json, but don't understand it's purpose.

File: src/firebase.js
Comment (pranati10) [reply src/firebase.js:1]: API key is changed to restrcited key by adding my IP address, so it is secured 
changes are made in the firebase console.

**Pull request closed without merging**